xbmc/addons/AddonDatabase.cpp

```diff
@@ -1284,3 +1284,32 @@ void CAddonDatabase::GetInstallData(const AddonInfoPtr& addon)
     CLog::Log(LOGERROR, "CAddonDatabase::{}: failed", __FUNCTION__);
   }
 }
+
+bool CAddonDatabase::AddInstalledAddon(const std::shared_ptr<CAddonInfo>& addon,
+                                       const std::string& origin)
+{
+  try
+  {
+    if (!m_pDB)
+      return false;
+    if (!m_pDS)
+      return false;
+
+    std::string now = CDateTime::GetCurrentDateTime().GetAsDBDateTime();
```

Comment:
As far as I remember, 'CDateTime::GetCurrentDateTime()' returns local time, not UTC. Is that intended?

Reply:
it is a copy from `SyncInstalled()` ;) so i didn't really question what it might return. please let me know, if (both) should change to something else

Reply:
IMO, change to UTC is a good idea, but if the field is stored in the db as local time elsewhere already, changing it would be semantically incompatible and db schema update incl. data migration is needed. Not sure this is worth the effort. Up to you.